src/renderer/captureTool/js/capture.js

```diff
@@ -0,0 +1,1014 @@
+"use strict";
+(function (fluid) {
+    var gpii = fluid.registerNamespace("gpii"),
+        ipcRenderer = require("electron").ipcRenderer;
+    fluid.registerNamespace("gpii.psp");
+    fluid.registerNamespace("gpii.captureTool");
+
+    var remoteFluid = require("electron").remote.getGlobal("fluid");
+
+    // TODO Ultimately these should be relocated to a JSON5 file, probably in
```

Comment:
Mention that such a JSON5 tool would constitute what Gregg is currently referring to as the "Presentation Registry"

Reply:
commented on